src/settings.py

```diff
@@ -28,6 +28,7 @@ class GlobalSettings:
     # cover_multistep_options parameters
     cover_multistep_action_limits = [-np.inf, np.inf]
     cover_multistep_use_learned_equivalents = True
+    cover_multistep_degenerate_oracle_samplers = False
```

Comment:
just curious what this setting does exactly?

Reply:
our usual sampler chooses a random pick or place anywhere on the block or target. if degenerate is True, then the sampler instead always chooses the midpoint, and there's no randomness in sampling at all.

ultimately we would expect this sampler to fail, but because of the way that we're generating the hand regions right now, it actually is expected to succeed always.